nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core/src/main/java/org/apache/nifi/controller/service/StandardControllerServiceNode.java

```diff
@@ -121,6 +122,11 @@ public Resource getResource() {
     }
 
     @Override
+    public boolean isRestricted() {
```

Comment:
Could this method be moved to `AbstractConfiguredComponent`? It seems like each place where it is implemented independently also extends that abstract class.